test/isInt.js

```diff
@@ -0,0 +1,40 @@
+var PushStateTree = require('../src/pushStateTree');
+
+describe('PushStateTree isInt', function() {
+
+  it('should return true for 0', function() {
+    expect(PushStateTree.isInt(0)).to.be.true;
+  });
+
+  it('should return true for 1', function() {
+    expect(PushStateTree.isInt(1)).to.be.true;
+  });
+
+  it('should return true for "0"', function() {
+    expect(PushStateTree.isInt('0')).to.be.true;
+  });
+
+  it('should return true for "1"', function() {
+    expect(PushStateTree.isInt('1')).to.be.true;
+  });
+
+  it('should return true for false', function() {
```

Comment:
nab: `should return false for Boolean values`
you can test for expect false and true in the same `it` block